src/core/server/graph/resolvers/Query.ts

```diff
@@ -21,4 +21,5 @@ export const Query: Required<GQLQueryTypeResolver<void>> = {
   debugScrapeStoryMetadata: (source, { url }, ctx) =>
     ctx.loaders.Stories.debugScrapeMetadata.load(url),
   moderationQueues: moderationQueuesResolver,
+  activeStories: (source, args, ctx) => ctx.loaders.Stories.activeStories(),
```

Comment:
Limit should be passed in from the provided arguments:

```suggestion
  activeStories: (source, { limit = 10 }, ctx) => ctx.loaders.Stories.activeStories(limit),
```